contrib/util/custom-firewall.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/env bash
 
+# obtain the etcd node members and check that at least there is three
+ETCD_NODES=$(curl -s http://localhost:4001/v2/members | jq '.[] | .[].peerURLs | length' | wc -l)
```

Comment:
`jq` is only on CoreOS hosts as of [717.0.0](https://coreos.com/releases/#717.0.0)

We'll have to wait until we bump hosts from 647.2.0 (hopefully soon!)